pydap_extras/app.py

```diff
@@ -1,19 +1,24 @@
-import sys
+import click
 from paste.httpserver import serve
 from pydap.wsgi.ssf import ServerSideFunctions
-from pydap_extras.handlers import csv, sql
 
-if __name__ == "__main__":
-    handler_type = sys.argv[1]
 
-    if handler_type == "csv":
-        csv._test()
-        application = csv.CSVHandler(sys.argv[2])
-    elif handler_type == "sql":
-        sql._test()
-        application = sql.CSVHandler(sys.argv[2])
-    else:
-        raise KeyError(f"No handler called {handler_type}")
+@click.command()
+@click.option("-h", "--handler-type")
+@click.option("-f", "--filename")
+@click.option("-p", "--port", default=8001)
```

Comment:
This is looking great! Just a couple things:
- added descriptions is really helpful for those that are not familiar with the options
- given that below you are using `handler_type` to define a module I would advise adding "choices"